Use GROUPS file from rpmlint (this project) package rather than rpm

--- rpmlint-rpmlint-1.9/Pkg.py~	2016-06-29 20:07:39.000000000 +0300
+++ rpmlint-rpmlint-1.9/Pkg.py	2016-08-06 12:10:45.102344608 +0300
@@ -214,16 +214,16 @@
 def get_default_valid_rpmgroups(filename=None):
     """
     Get default rpm groups from filename, or try to look them up from
-    the rpm package (if installed) if no filename is given.
+    the rpmlint package (if installed) if no filename is given.
     """
     groups = []
     if not filename:
         try:
-            with InstalledPkg("rpm") as p:
+            with InstalledPkg("rpmlint") as p:
                 groupsfiles = [x for x in p.files() if x.endswith('/GROUPS')]
                 if groupsfiles:
                     filename = groupsfiles[0]
-        except KeyError:  # the rpm package might not be installed
+        except KeyError:  # the rpmlint package might not be installed
             pass
     if filename and os.path.exists(filename):
         with open(filename) as fobj:
